### PR TITLE
build: fix broken symbols test

### PR DESCRIPTION
**Pull request by crisbeto**
Fixes a test that is broken on the main branch.

### CODE REVIEW

File: packages/core/test/bundling/defer/bundle.golden_symbols.json [hunk 1]
```diff
@@ -1802,6 +1808,9 @@
   {
     "name": "init_signal"
   },
+  {
+    "name": "init_signal2"
+  },
```
Comment (devversion) [inline packages/core/test/bundling/defer/bundle.golden_symbols.json:1813]: does this indicate code duplication?

Comment (crisbeto) [reply packages/core/test/bundling/defer/bundle.golden_symbols.json:1813]: Potentially 🤔. I'm mostly trying to fix the CI before more PRs come in.

**Review by AndrewKushnir — APPROVED** [review]

File: packages/core/test/bundling/defer/bundle.golden_symbols.json (hunk 1 above)
Comment (devversion) [reply packages/core/test/bundling/defer/bundle.golden_symbols.json:1813]: We should create an issue for tracking I think

**Comment by crisbeto** [comment]
Caretaker note: for some reason the `merge status` check refers to a job that no longer exists. It's unrelated to these changes.

**Comment by atscott** [comment]
This PR was merged into the repository by commit 753eb2459ed7e44f265b8a522a48155ea84a56b9.

**Pull request closed without merging**